Sming/rboot/Makefile

```diff
@@ -37,6 +37,9 @@ endif
 ifeq ($(RBOOT_INTEGRATION),1)
 	CFLAGS += -DRBOOT_INTEGRATION
 endif
+ifeq ($(RBOOT_RTC_ENABLED),1)
```

Comment:
rBoot is an external component included in sming unmodified, please supply a patch upstream to rBoot and I'll get it merged